tests/routers/test_tags_security.py

```
@@ -0,0 +1,74 @@
+from fastapi.testclient import TestClient
+from app.main import app
+from app.security import get_current_user
+
+class MockUser:
+    id = 1
+    username = "admin"
+    roles = ["admin"]
```

Ruff (0.15.1)

[warning] Mutable default value for class attribute

(RUF012)